kas/iot2050.yml

```diff
@@ -27,16 +27,9 @@ repos:
 
   isar:
     url: https://github.com/ilbers/isar
-    refspec: 74036ebaab5265d62cdf90f563acd5b4d3b8b6d2
+    refspec: cf3f18a649ff42726677a58b224b9bb58ea4ba46
     layers:
       meta:
-    patches:
-      deb-build-profiles:
-        repo: meta-coral
-        path: patches/isar/0001-add-support-for-debian-build-profiles-and-options.patch
-      adjust-kernel-recipe:
-        repo: meta-coral
-        path: patches/isar/0002-refactor-linux-custom.inc-to-use-ISAR-s-DEB_BUILD_PR.patch
```

Comment:
This patch likely has to come before the meta-coral dump, see my earlier comment.